serve/tests/unittest/test_sampler.py

```diff
@@ -122,35 +158,47 @@ def _test_penalties_checker():
         get_sampling_metadata([SamplingParams(frequency_penalty=-2.1)])
 
     with pytest.raises(ValueError):
-        get_sampling_metadata([SamplingParams(repetition_penalty=-2.1)])
+        get_sampling_metadata([SamplingParams(repetition_penalty=0.0)])
 
     with pytest.raises(ValueError):
         get_sampling_metadata([SamplingParams(presence_penalty=2.1)])
 
     with pytest.raises(ValueError):
         get_sampling_metadata([SamplingParams(frequency_penalty=2.1)])
 
+    with pytest.raises(ValueError):
```

Comment:
What should this test check and several ones below?

Reply:
A simple check to make sure that different results in different queries do not affect each other